src/main/scala/zio/prelude/Equal.scala

```diff
@@ -803,6 +828,15 @@ object Equal extends Lawful[Equal] {
         a1 === a2 && b1 === b2 && c1 === c2 && d1 === d2 && e1 === e2 && f1 === f2 && g1 === g2 && h1 === h2 && i1 === i2 && j1 === j2 && k1 === k2 && l1 === l2 && m1 === m2 && n1 === n2 && o1 === o2 && p1 === p2 && q1 === q2 && r1 === r2 && s1 === s2 && t1 === t2 && u1 === u2 && v1 === v2
     }
 
+  /**
+   * Equality for `Throwable` values.
+   * Comparison is based on: Class, message and cause (stack trace is ignored).
+   */
+  implicit val ThrowableEqual: Equal[Throwable] =
```

Comment:
Not quite sure what the right answer is here. We probably need some instance for derived types that use `Throwable` but the exact definition is not as clear cut as I would like.

Reply:
Yeah, this is something I've been discussing with @jdegoes, so John what do you think about this definition?